CHANGELOG.md

```diff
@@ -3,17 +3,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
-## [2.2.x] - ???
+## [2.2.4] - 2023-08-??
 ### :magic_wand: Added
 - Host Availability Strategy to help keep host health status up to date ([PR #530](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/530)).
+- Implement `setLoginTimeout` from a suggested enhancement ([Discussion #509](https://github.com/awslabs/aws-advanced-jdbc-wrapper/discussions/509)).
 
 ### :crab: Changed
 - Dynamically sets the default host list provider based on the dialect used. User applications no longer need to manually set the AuroraHostListProvider when connecting to Aurora Postgres or Aurora MySQL databases.
 - Deprecated AuroraHostListConnectionPlugin. 
   - As an enhancement, the wrapper is now able to automatically set the Aurora host list provider for connections to Aurora MySQL and Aurora PostgreSQL databases.
     Aurora Host List Connection Plugin is deprecated. If you were using the `AuroraHostListConnectionPlugin`, you can simply remove the plugin from the `wrapperPlugins` parameter.
     However, if you choose to, you can ensure the provider is used by specifying a topology-aware dialect, for more information, see [Database Dialects](docs/using-the-jdbc-driver/DatabaseDialects.md).
+- Propagate `Connection.clearWarnings()` to underlying connections so that the connection object does not accumulate warning messages  ([Issue #547](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/547)).
```

Comment:
```suggestion
- Propagate `Connection.clearWarnings()` to underlying connections in the Read Write Splitting Plugin so that the connection object does not accumulate warning messages  ([Issue #547](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/547)).
```

Reply:
Done!